sdk/process/keeper.go

```diff
@@ -6,21 +6,21 @@ import (
 	cosmostypes "github.com/cosmos/cosmos-sdk/types"
 	"github.com/mesg-foundation/engine/codec"
 	"github.com/mesg-foundation/engine/hash"
-	"github.com/mesg-foundation/engine/ownership"
+	ownershippb "github.com/mesg-foundation/engine/ownership"
```

Comment:
As said in https://github.com/mesg-foundation/engine/issues/1639#issuecomment-584042783, we can also move the proto go struct directly in `x/module/ownership` package or sub-package:
>  execution, instance, ownership, process, runner, service packages should move to x/ packages. All of the code is about data definition or validation.

Reply:
I didn't want to put that in this PR, but I can if you want to.